### PR TITLE
FIX #182 - Modified add operator to allow the user to enter a well-known name.

### LINKED ISSUE #182: add operator using well known name
Well known operators are baked into the app or set in the env via NSC_name_OPERATOR.
To simplify the setup for managed operators like ngs, using a well know operator name saves the user from typing a long URL.

Instead of specifying a file/url allow the url field to have the 'name' of the well known operator.

### PR DESCRIPTION
**Pull request by aricart**

### CODE REVIEW

File: cmd/addoperator.go [hunk 1]
```diff
@@ -87,7 +88,21 @@ func (p *AddOperatorParams) PreInteractive(ctx ActionCtx) error {
...
-		p.jwtPath, err = cli.Prompt("path or url for operator jwt", p.jwtPath, true, cli.PathOrURLValidator())
+		p.jwtPath, err = cli.Prompt("path or url for operator jwt", p.jwtPath, true, func(v string) error {
+			// is it is an URL or path
+			pv := cli.PathOrURLValidator()
+			if err := pv(v); err != nil {
+				// if it doesn't exist - could it be the name of well known operator
+				if os.IsNotExist(err) {
+					wko, _ := FindKnownOperator(v)
+					if wko != nil {
+						return nil
```
Comment (kozlovic) [inline cmd/addoperator.go]: This does not make sense to me. If `wko` is not nil, then we return `nil` (for no error), but do the same if it is `nil`..

File: cmd/addoperator.go [hunk 2]
```diff
@@ -87,7 +88,21 @@ func (p *AddOperatorParams) PreInteractive(ctx ActionCtx) error {
...
+					}
+					return nil
+				}
+			}
+			return nil
```
Comment (kozlovic) [inline cmd/addoperator.go]: So we never return an error in this validation function..

File: cmd/addoperator.go (hunk 1 above)
Comment (aricart) [reply cmd/addoperator.go]: it is a validator - there's no error in that case.

Comment (kozlovic) [reply cmd/addoperator.go]: Ok, but look at the code, this does not make sense. You return the same result (nil) regardless if wko is nil or not.

File: cmd/addoperator.go (hunk 2 above)
Comment (aricart) [reply cmd/addoperator.go]: yes this last return is wrong, it needs to return err at that point.

File: cmd/addoperator.go [hunk 3]
```diff
@@ -87,8 +88,20 @@ func (p *AddOperatorParams) PreInteractive(ctx ActionCtx) error {
...
+			if err := pv(v); err != nil {
+				// if it doesn't exist - could it be the name of well known operator
+				if os.IsNotExist(err) {
+					wko, _ := FindKnownOperator(v)
+					if wko != nil {
+						return nil
+					}
+				}
+			}
+			return err
```
Comment (kozlovic) [inline cmd/addoperator.go]: This is wrong too. Here should be probably return `nil`, but you should have a `return err` at line 101. That is, return the error from pv(v). Currently you are returning `err` that is defined before, say line 91.

**Pull request merged**